Analyses/src/BFieldPlotter_module.cc

```diff
@@ -137,76 +137,76 @@ namespace mu2e {
     //check that the map bin step size works with the edges given
     if(nbinsOne > 1 && std::abs(_axisOneMin + (nbinsOne-1)*_mapBinSize - _axisOneMax > (_axisOneMax-_axisOneMin)/(100.*(nbinsOne-1))))
       throw cet::exception("BADCONFIG") << "BField mapping axis values not steppable with step size given: "
-					<< _axisOneMin << " < axisOneValues < " << _axisOneMax << ", "
-					<< "step size = " << _mapBinSize;
+        << _axisOneMin << " < axisOneValues < " << _axisOneMax << ", "
+        << "step size = " << _mapBinSize;
 
     if(nbinsTwo > 1 && std::abs(_axisTwoMin + (nbinsTwo-1)*_mapBinSize - _axisTwoMax > (_axisTwoMax-_axisTwoMin)/(100.*(nbinsTwo-1))))
       throw cet::exception("BADCONFIG") << "BField mapping axis values not steppable with step size given: "
-					<< _axisTwoMin << " < axisTwoValues < " << _axisTwoMax << ", "
-					<< "step size = " << _mapBinSize;
+        << _axisTwoMin << " < axisTwoValues < " << _axisTwoMax << ", "
+        << "step size = " << _mapBinSize;
 
     //if a null map, plot the default field from the manager
     const std::string name = (map) ? map->getKey() : "default";
     if(_hMap[name]) return; //already filled the maps in a previous event
 
     // if dumping, create the file
-    std::fstream fs;
+    std::ofstream fs;
     if(_dump){
       string dumpfile= name+_dumpname+".csv";
       fs.open (dumpfile.c_str(), fstream::out);
       if(fs.is_open()){
-	fs << "# Dump of " << name << endl;
-	fs <<  "#  x   ,  y   ,   z   ,  Bx   ,  By   ,  Bz" << endl;
-      }
-
+        fs << "# Dump of " << name << endl;
+        fs <<  "#  x   ,  y   ,   z   ,  Bx   ,  By   ,  Bz" << endl;
+      } else
+        throw cet::exception("BADCONFIG") << "Cannot open file " << dumpfile << endl;
```

Comment:
Can we please add braces after the "else"?

Reply:
does this code produce erroneous results?  is it incorrectly indented?  Does it violate Mu2e coding standards?  If not, why are you requesting a change?

Reply:
I think a Mu2e coding standard doc asked for braces.   I can't locate that docdb at the moment, but I think it makes sense.  Especially in the current case where the "if" has braces -  "else" without them looks weird.

Reply:
Everything we had for coding standards is [here](https://mu2ewiki.fnal.gov/wiki/CodingStandards) and links therein. Maybe you are thinking of [this doc](https://cdcvs.fnal.gov/redmine/attachments/download/19145/CodingStandards.pdf).  After a quick glance through, I didn't find this question addressed.  If there is another doc, let me know so I can link it.

Reply:
I have a preference for braces but I don't believe we never formalized it because it was too contentious.